websites/H/HackTheBox/presence.ts

```diff
@@ -0,0 +1,197 @@
+const presence = new Presence({
+    clientId: "834093191485063258"
+});
+
+var browsingStamp = Math.floor(Date.now() / 1000);
+
+presence.on("UpdateData", () => {
+    var presenceData: PresenceData = {
+      largeImageKey: "logo"
+};
+
+presenceData.startTimestamp = browsingStamp;
+
+const getElement = (query: string): string | undefined => {
+    return document.querySelector(query) ? document.querySelector(query).textContent: undefined;
+};
+
+function capLet(string) {
+    return string.charAt(0).toUpperCase() + string.slice(1);
+}
```

Comment:
first you use arrow function then normal? ;-; Move it outside presence.on event.
and technically getElement will not work like it should - why? Because when it will return undefined it will show for example `Viewing undefined's Profile` I dont think it should be like this :/
```suggestion
const getElement = (query: string): string | undefined => {
    return document.querySelector(query) ? document.querySelector(query).textContent : undefined;
}, capLet = (text: string): string => {
    return text.charAt(0).toUpperCase() + text.slice(1);
};
```